android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/MapCompat.java

```diff
@@ -10,8 +10,13 @@
  * Reference: https://stackoverflow.com/questions/41211960/alternative-to-getordefault-for-devices-below-api-24-android
  */
 public final class MapCompat {
+  private static final String TAG = "MapCompat";
 
   public static <K, V> V getOrDefault(HashMap<K, V> map, K key, V defaultValue) {
+    if (map == null) {
```

Comment:
Again, I think this test should be in the caller, not in the utility function. This is called by `KeyboardSettingsActivity.onItemClick` (:165) and it seems that in some circumstances, there may not be a hash map associated with the clicked item -- can we figure out why?

Reply:
I've moved the check to KeyboardSettingsActivity.onItemClick and added logging. Hopefully that will give us insight in Sentry how the hash map is getting null